runtime/native/jvm.cpp

```diff
@@ -539,6 +539,9 @@ jclass JVM_FindClassFromBootLoader(JNIEnv *env, const char *name) {
 
 jclass JVM_FindClassFromCaller(JNIEnv *env, const char *name, jboolean init,
                                jobject loader, jclass caller) {
+    // TODO: GLOBALMUTEX
```

Comment:
Is this TODO still accurate? Is the line after this not acquiring the gobal mutex?
(same goes for the other comments like this one)

Reply:
No, it just reminded me where we used the globalMutex. I believed we could remove these TODOs now.